### PR TITLE
NOTIF-126 add build_deploy script

### PR DESCRIPTION
**Pull request by lahavyuv86**

### CODE REVIEW

File: build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE="quay.io/notifications-backend"
```
Comment (theute) [inline build_deploy.sh:5]: I guess it's missing the group name. Which I guess is still `cloudservices` ?

File: build_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+mvn clean package \
+    -DskipTests
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
+
+docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
+docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" .
```
Comment (theute) [inline build_deploy.sh:26]: If you use `docker` and not benefit from the Quarkus plugin, I guess it won't find the Dockerfile which lives in `src/main/docker/Dockerfile.jvm`

**Review by theute — CHANGES_REQUESTED** [review]

File: build_deploy.sh (hunk 1 above)
Comment (lahavyuv86) [reply build_deploy.sh:5]: right.. will fix it.

File: build_deploy.sh (hunk 2 above)
Comment (lahavyuv86) [reply build_deploy.sh:26]: `src/main/docker/Dockerfile.jvm` was used via Quarkus, when testing it with docker it looks for Dockerfile at the same path the build_script is located.

Comment (theute) [reply build_deploy.sh:26]: I missed the file rename/move in the change log, all good

**Review by theute — APPROVED** [review]

**Pull request merged**